src/main/kotlin/it/pagopa/wallet/scheduler/documents/Client.kt

```diff
@@ -0,0 +1,3 @@
+package it.pagopa.wallet.scheduler.documents
+
+data class Client(val status: String, val lastUsage: String?)
```

Comment:
last usage field have been removed from wallet service by _old update wallet logic refactoring_ and have been never written into prod env. `lastUsage` can be safely been removed here

Reply:
done here 229d5d5c359798a25d917e4eac2eb4e4be5f51ca